bin/variety

```diff
@@ -56,6 +56,16 @@ if os.path.abspath(__file__).startswith("/opt"):
 if os.path.exists(os.path.join(PROJECT_ROOT_DIRECTORY, "variety")):
     python_path.insert(0, PROJECT_ROOT_DIRECTORY)
     sys.path.insert(0, PROJECT_ROOT_DIRECTORY)
+elif ".local" in PROJECT_ROOT_DIRECTORY:
+    for python_version in [3.5, 3.6, 3.7, 3.8]:
```

Comment:
Probably better to read the current Python version from `sys.version_info` instead of hardcoding a list of versions here.

Reply:
You are right, I change it, and rebased (and re-worded the git commit).